clang/test/CIR/CodeGen/tbaa.c

```diff
@@ -0,0 +1,121 @@
+// modified from clang/test/CodeGen/tbaa.c
```

Comment:
Thanks for the comprehensive tests! Can you please rename them to be a bit more descriptive? Example: if this is only dealing with enums, should `tbaa-enum.c`. Also, if the test is only checking for LLVM output, it should be under `Lowering`, if it's testing both CIR and LLVM, than `CodeGen` is good enough.